test/fwiGradientTest.jl

```diff
@@ -8,6 +8,8 @@
 
 using JUDI.TimeModeling, Test, LinearAlgebra, PyPlot, Printf
 
+include("test_utils.jl")
```

Comment:
No, this is already included in `runtest.jl` and is not needed if test is run as it's supposed to in julia.